aszczachor/src/components/mainPage/MainContent.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const MainContent = () => {
+  return <div>Tu gridzik z fotosami</div>;
```

Comment:
Nie wrzucałbym takich rzeczy na przyszłość, to taki trochę komentarz, który w tym momencie nie przynosi żadnej wartości, generalnie dodałbym kawałek po kawałku zamiast rzeczy, które na ten moment nie przydadzą się